indra/databases/pubchem_client.py

```diff
@@ -74,3 +75,28 @@ def get_preferred_compound_ids(pubchem_cid):
                 pref_ids |= set(pref_cpd['Value']['Number'])
     pref_ids = sorted([str(pid) for pid in pref_ids])
     return pref_ids
+
+
+def get_pmids(pubchem_cid: str) -> List[str]:
+    """Return xref PMIDs for a given PubChem CID.
+
+    Parameters
+    ----------
+    pubchem_cid :
+        The PubChem CID whose xref PubMedID's will be returned.
+
+    Returns
+    -------
+    list of str
```

Comment:
```suggestion
    :
```

don't forget this ^ it's important to make sure the doc strings don't contain these so they don't get out of sync and so they're properly programatically parsed. The `:` is the placeholder

Reply:
I agree, I changed this